jplag.endToEndTesting/src/main/java/model/ResultJsonModel.java

```diff
@@ -0,0 +1,31 @@
+package model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class ResultJsonModel {
+
+	@JsonProperty("functionName")
```

Comment:
json is typically more like "function_name" afaik

Reply:
any thoughts @tsaglam @JanWittler @sebinside

Reply:
We should keep it somewhat consistent with the JSON we generate for the report viewer, I would say.

Reply:
I'll take a look at it. I have to adjust the json format anyway to allow tests that have more than one result. But I had planned to deal with this in another PR where the extension with the storage is also included.

Reply:
changed naming to "result_similarity" and "function_name" in ResultJsonModel and json file

``` java 
	@JsonProperty("function_name")
	private String functionName;
	@JsonProperty("result_similarity")
	private String resultSimilarity;
```